lib/libutee/include/pta_versal_fpga.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2023, ProvenRun SAS
+ */
+
+#ifndef __PTA_VERSAL_FPGA_H
+#define __PTA_VERSAL_FPGA_H
+
+#define PTA_VERSAL_FPGA_UUID { 0xa6b493c0, 0xe100, 0x4a13, \
+	{ 0x9b, 0x00, 0xbc, 0xe4, 0x2d, 0x53, 0xce, 0xd8 } }
+
+/**
+ * Write FPGA bitstream
+ *
+ * [in]		memref[0].buffer	FPGA bitstream buffer
+ * [in]		memref[0].size		FPGA bitstream buffer size
```

Comment:
usual description would rather be a single line:
```
 * [in]		memref[0]	FPGA bitstream buffer

Reply:
Fixed in the latest push.